### PR TITLE
.coafile: Add YAMLLintBear

### PR DESCRIPTION
**Pull request by yagyanshbhatia**
Add YAMLLintBear in .coafile and Fixed the issues raised by coala
Closes https://github.com/coala/coala-vs-code/issues/14

### CODE REVIEW

**Comment by yagyanshbhatia** [comment]
@Monal5031 @Makman2  is this PR acceptable now?

**Comment by yagyanshbhatia** [comment]
okay, changed it. anything else @newbazz 😄

**Comment by ishanSrt** [comment]
unack 511b4aec3307cca9874196477f6226309194a980

**Comment by yagyanshbhatia** [comment]
Its already there :/
Closes #14
That's why this issue got referenced here :)
@ishanSrt

**Comment by ishanSrt** [comment]
@yagyanshbhatia I am talking about your commit not the comment on your PR

**Comment by ishanSrt** [comment]
Also it seems you have two PRs open to fix the same issue. Please close the other one

**Comment by ishanSrt** [comment]
@yagyanshbhatia Close https://github.com/coala/coala-vs-code/pull/21
after that use `git commit --amend` and have your commit message follow these guidelines: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html 
After that you can force push to this branch

**Comment by ishanSrt** [comment]
I see you have added YAMLLintBear in .coafile but what are the issues raised my coala that you fixed?

**Comment by yagyanshbhatia** [comment]
@ishan I got your first point of closes statement, I am updating it
I was told to squash commits into one so should I just explain these issues in comment on the PR? OR explain in commit message instead?

**Comment by ishanSrt** [comment]
@yagyanshbhatia commit message is only to deal with the changes you are doing, while comments are to improve your PR, the commit message and the changes you have done

**Comment by ishanSrt** [comment]
You were told to squash the commits but you did not do that, instead you opened another PR for that with a single commit. You should have done that on the same PR, but don't worry. Just don't open more PRs. You can always change your existing PRs. And close the other one.

**Comment by yagyanshbhatia** [comment]
@ishanSrt okay i closed the older commit, and changed the commit message body. Is there anything else i can do? sorry for being so naive, i am totally new to these things 😄 .

**Comment by ishanSrt** [comment]
@yagyanshbhatia how do you reproduce the issues raised by coala?

**Comment by yagyanshbhatia** [comment]
if we'll add trailing spaces at the end of the line, spaceconsistencybear will give error
if length of any line exceeds 79, it will give an error. Is that what you meant?

**Comment by ishanSrt** [comment]
@yagyanshbhatia Please look at the the reviews I have made. Also I don't think that an issue should ask to solve more than one problem. Adding YAMLLintBear to .coafile is one thing, and fixing other inconsistencies like spaces and all should be a separate issue. Ping a maintainer to change the issue and open a different issue for the other problems

**Comment by yagyanshbhatia** [comment]
okay @ishanSrt . thank you for helping this much.
@Makman2 please look into what ishan suggested..?

**Comment by ishanSrt** [comment]
@yagyanshbhatia still look at the code reviews I made eg. there should be a newline at the end of file. I think gitmate bot points that out.

**Comment by ishanSrt** [comment]
@yagyanshbhatia also I see a PR already open for this issue before you started working on this

**Comment by yagyanshbhatia** [comment]
Yes, even I reported that. But makman said it's something different, you carry on with this issue. I think he worked on it without being assigned.

**Comment by Monal5031** [comment]
@yagyanshbhatia Why did you close the previous PR, if you were having trouble with git you could've asked on gitter channel for help. The whole point of doing newcomer issue to learn git and to teach the way things work in coala. Closing PRs won't help in any way.

File: .coafile [hunk 1]
```diff
@@ -39,3 +39,7 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+[yml]
+bears = YAMLLintBear
+files = *.yml
```
Comment (Monal5031) [inline .coafile]: newline here.

Comment (yagyanshbhatia) [reply .coafile]: @Monal5031 I added newline at the end of .coafile :)
i understand, i'll never open new PRs in future, is there anything else how this PR can be improved?

Comment (Monal5031) [reply .coafile]: Yeah the changes look good, but I just noticed that we aren't runnig coala on travis. Should we make that change in this PR itself @Makman2 ?

Comment (Makman2) [reply .coafile]: No that's okay, not task of this issue :+1:

**Comment by yagyanshbhatia** [comment]
Can we merge it now??

File: .travis.yml [hunk 2]
```diff
@@ -30,7 +30,8 @@ before_script:
   # nltk 3.2.2 dropped support for Python 3.3
   - >
     if [[ "$TRAVIS_PYTHON_VERSION" != "3.3" ]]; then
-      python -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
+      python -m nltk.downloader \
+      punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
```
Comment (Makman2) [inline .travis.yml]: I would indent by another 2 spaces so someone can immediately see that those two lines have a relationship ;)

**Comment by Makman2** [comment]
unack 07054e5

File: .travis.yml (hunk 2 above)
Comment (yagyanshbhatia) [reply .travis.yml]: @Makman2 you mean another two whitespaces before 'punkt' or both the lines?

Comment (Makman2) [reply .travis.yml]: just before `punkt` :)

Comment (yagyanshbhatia) [reply .travis.yml]: done :) @Makman2 now please merge it sir, i am starving to move on to another one 😄

**Comment by Makman2** [comment]
ack 3a34070

**Comment by Makman2** [comment]
@gitmate-bot ff

**Comment by Makman2** [comment]
ack cc2cd9b

**Comment by Makman2** [comment]
oops wrong commit before...

**Pull request merged**